en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -55,6 +55,33 @@ Conducting security vulnerability scans is crucial for identifying potential wea
 
 During deployment, Choreo performs a Trivy scan to detect critical vulnerabilities in third-party libraries added to the integration component. If the scan detects any critical vulnerabilities, it halts the deployment process. The deployment pane shows the Trivy scan status and any security failures in the Library (Trivy) vulnerable scan step, which you can access by clicking on it. After fixing the vulnerability, you can redeploy the component.
 
+## Configuring Micro Integrator
+
+WSO2 MI comes with a set of pre-configured settings by default, designed to work in a general context. However, these default configurations might not be suitable for every organization's specific needs. As a result, it is crucial to modify the default configuration in WSO2 MI to optimize its performance and ensure compatibility with an organization's systems and applications.
+
+To customize the default configurations in WSO2 MI, users can apply their desired configurations to the instances running on Choreo. To add new configurations, users need to define a `deployment.toml` file in the project sub-path. Similar to configuring MI locally, users can apply configuration changes in the `deployment.toml` file. However, it is essential to apply only the required changes to avoid disrupting internal communication. It's worth noting that altering default configurations such as port offset, hostname, and other critical settings might break internal communication. Therefore, it's advisable to avoid modifying them unless necessary.
+
+Below is a sample `deployment.toml` configuration for configuring JMS transport. Refer to
```

Comment:
Suggestion:

Here is a sample `deployment.toml` file that can be used to configure the JMS transport. For more information on WSO2 MI configuration parameters, see the [MI Config Catalog](https://apim.docs.wso2.com/en/latest/reference/config-catalog-mi/).